pom.xml

```diff
@@ -59,47 +59,54 @@
       <artifactId>gson</artifactId>
       <version>2.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>me.moocar</groupId>
+      <artifactId>socket-encoder-appender</artifactId>
+      <version>0.1beta1</version>
+    </dependency>
```

Comment:
yeah, I'm leaning towards including it now since It's so small. Btw, I did include the [dependency information](https://github.com/Moocar/logback-gelf#tcp) in the README.

Thanks again for the PR